src/norwegianblue/cli.py

```diff
@@ -161,7 +161,8 @@ def main() -> None:
                 product=suggestion,
                 format=args.formatter,
                 color=args.color,
-                show_title=multiple_products,
+                show_title=(args.show_title == "yes")
+                or (multiple_products and args.show_title != "no"),
```

Comment:
Let's define this right after `multiple_products` above:

```python
    multiple_products = len(args.product) >= 2
    show_title=(args.show_title == "yes") or (multiple_products and args.show_title != "no")
```

And then use this in both places:

```suggestion
                show_title=show_title,
```